prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
```

Comment:
Fuel has a little comment about what it is, name and odometer do not.

Reply:
Noted and amended, thank you.